src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -364,6 +373,38 @@ def get_command(self, additional_args=None):
     # list" and then run that fuzzer.
     return self.ssh_command('ls')
 
+  def fetch_and_process_logs_and_crash(self):
+    """Fetch symbolized logs and crashes."""
+    # Get the crash from the device.
+    self.device.fetch(
+        self.fuzzer.data_path('crash*'), self.fuzzer.results_output())
+
+    for logname in os.listdir(self.fuzzer.results_output()):
+      if logname in self.fuzzer.logfile:
+        self.device.dlog(self.fuzzer.logfile)
+
+    # Clusterfuzz assumes that the Libfuzzer output points to an absolute path,
+    # where it can find the crash file.
+    # This doesn't work in our case due to how Fuchsia is run.
+    # So, we make a new file, change the appropriate line with a regex to point
+    # to the true location. Apologies for the hackery.
+    crash_testcase_file_path = ''
+    crash_location_regex = r"(.*)(Test unit written to )(data/.*)"
+    _, new_file_handle_path = tempfile.mkstemp()
+    with open(new_file_handle_path, 'w') as new_file:
+      with open(self.fuzzer.logfile) as old_file:
+        for line in old_file:
+          line_match = re.match(crash_location_regex, line)
+          if line_match:
+            crash_testcase_file_path = os.path.join(
+                self.fuzzer.results_output(),
+                line_match.group(3).replace('data/', ''))
+          new_text = re.sub(crash_location_regex,
```

Comment:
now that the code has been re-structured, we can indent this bit and just do:

`line = re.sub(crash_location_regex, ..., line`

This way we can get rid of line 391, and the `new_text` variable.